airbyte-integrations/connectors/source-aha/source_aha/manifest.yaml

```diff
@@ -102,8 +102,8 @@ streams:
 
 check:
   stream_names:
-    - "features"
     - "products"
+    - "features"
     - "ideas"
     - "users"
     - "goals"
```

Comment:
@Upgreydd you want to remove the goals and features from check connection?

Reply:
@marcosmarxm looks like connection check base only on first element, when I changed order from optional `features` first to `products` - then it pass.